third_party/winuwp_h264/H264Encoder/H264Encoder.cc

```diff
@@ -43,6 +43,11 @@ namespace webrtc {
 // QP scaling thresholds.
 static const int kLowH264QpThreshold = 24;
 static const int kHighH264QpThreshold = 37;
+
+// On some encoders (e.g. Hololens 2) changing rates is slow so we don't want to
+// do it too often.
+static const int kMinIntervalBetweenRateChanges = 1000;
```

Comment:
>1000 [](start = 50, length = 4)

Since this varies based on the hw+driver combination, how about reading this value from registry? In that case we don't need to penalize hw+drivers that change rates fast.

Reply:
Logged as https://github.com/webrtc-uwp/webrtc-uwp-sdk/issues/206